CONTRIBUTING.md

```diff
@@ -3,20 +3,14 @@
 Setup quickly a development environment with Docker & Compose.
 
 ``` console
-$ wget https://raw.githubusercontent.com/dalibo/docker/master/temboard/docker-compose.yml
-$ docker-compose up
+$ mkvirtualenv temboard
```

Comment:
I would replace this by something saying "Create a python virtualenv and activate it"